doc/isomorph.xml

```diff
@@ -811,3 +811,30 @@ gap> OutNeighbours(canon);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="IsDigraphAutomorphism">
+<ManSection>
+  <Oper Name="IsDigraphAutomorphism" Label="for a digraph and permutation"
+    Arg="digraph, x"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description> 
+    This operation returns true if the permutation <A>x</A> is an automorphism
+    of the digraph <A>digraph</A>. An permutation <C>g</C> on the vertices of a
+    digraph <C>gr</C> is an automorphism if it satisfies: <C>(u, v)</C> is an
+    edge of <C>gr</C> if and only if <C>(g(u), g(v))</C> is an edge of
+    <C>gr</C>. See also <Ref Attr="AutomorphismGroup" Label="for a digraph" />.
```

Comment:
Please change it to:

...of `<C>gr</C>` for all vertices <C>u</C> and <C>v</C> in <C>digraph</A>. (To quantify the vertices)